financial-connections/src/main/java/com/stripe/android/financialconnections/features/manualentry/ManualEntryScreen.kt

```diff
@@ -102,10 +102,8 @@ private fun ManualEntryContent(
     ) {
         when (payload) {
             is Loading, Uninitialized -> FullScreenGenericLoading()
-            is Fail -> ErrorContent(
+            is Fail -> UnclassifiedErrorContent(
```

Comment:
is this change related to the shared state?

Reply:
it is - noticed that an the Partner Auth error screen was used on the manual entry pane. This error screen has custom views for specific partner auth errors, and falls back to `UnclassifiedErrorContent`. 

In manual entry we don't have custom error screens, `UnclassifiedErrorContent` should be displayed on any error.